src/v/pandaproxy/schema_registry/service.cc

```diff
@@ -188,6 +191,45 @@ ss::future<> service::do_start() {
     });
 }
 
+ss::future<>
+create_acls(cluster::security_frontend& security_fe, ss::abort_source& as) {
+    // Initialize with any non-success errc
+    std::vector<cluster::errc> err_vec{cluster::errc::timeout};
+
+    auto create_acls_backoff = initial_create_acls_backoff;
+    std::vector<security::acl_binding> princpal_acl_binding{
+      security::acl_binding{
+        security::resource_pattern{
+          security::resource_type::topic,
+          model::schema_registry_internal_tp.topic,
+          security::pattern_type::literal},
+        security::acl_entry{
+          principal,
+          security::acl_host::wildcard_host(),
+          security::acl_operation::all,
+          security::acl_permission::allow}}};
+
+    while (err_vec[0] != cluster::errc::success) {
+        err_vec = co_await security_fe.create_acls(princpal_acl_binding, 1s);
+
+        if (err_vec[0] != cluster::errc::success) {
+            vlog(
+              plog.trace,
+              "Failed to create ACLs for {}, err {} - {}",
```

Comment:
should we address the scenario of someone being able to debug this without resorting to turning on full trace-level logging?

Reply:
I've gone back-and-forth in my head on making this log debug, warn, or trace. I think you are correct that trace is too severe though.

Reply:
I don't think in this case that you need to choose between a set of possible trace levels. Consider the cases:

1. we trace log the result of every attempt to create acls. this is the backup plan for debugging to turn on trace logging which is usually a big hammer because of how much data it produces.
2. now consider the failure case. the loop is infinite, and I'm assuming the schema registry doesn't work unless creating the acls succeeds? this seems like a scenario where we would want to log at info/warn: hey, things aren't going well here.
3. so one way to do this without having to spam the log is to use seastar::logger::rate_limit feature which lets you have a logging point which only prints out a message like once every 30 seconds or something.
4. so you could print the failure message early on (say if it didn't succeeded in the first few attempts), and then after that only log the error infrequently.